### PR TITLE
Elasticsearch in out example code

### PR DESCRIPTION
**Pull request by KieranCTaylor**
While playing with scio-elasticsearch I noticed there weren't any documented example so added a toy example of how to establish connection and index a json document

### CODE REVIEW

File: scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    val primaryHost = new HttpHost(host, port)
+    val servers: Seq[HttpHost] = Seq(primaryHost)
+
+    val clusterOpts = ElasticsearchOptions(servers)
+
+    // Provide an elasticsearch writer to transform collections to es documents
+    val indexWriter = indexer(index)
+
+    // Read a simple text file and write the first 10 characters to an elasticsearch document
+    ElasticsearchSCollection(sc.textFile(input))
```
Comment (regadas) [inline scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala]:
```suggestion
   sc.textFile(input)
```

File: scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+// --input=[INPUT] --index=[INDEX] --esHost=[HOST] --esPort=[PORT]"`
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio._
+import com.spotify.scio.elasticsearch._
+
+import org.apache.http.HttpHost
+import org.elasticsearch.action.index.IndexRequest
+
+object ElasticsearchInOut {
```
Comment (regadas) [inline scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala]: 🎨 `ElasticsearchMinimalExample`

File: scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Example: Elasticsearch Input and Output
```
Comment (regadas) [inline scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala]: Can you expand this doc a little bit on what the example does?

File: scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    // Output es index to write into
+    val index = args.getOrElse("index", "defaultIndex")
+
+    val primaryHost = new HttpHost(host, port)
+    val servers: Seq[HttpHost] = Seq(primaryHost)
+
+    val clusterOpts = ElasticsearchOptions(servers)
+
+    // Provide an elasticsearch writer to transform collections to es documents
+    val indexWriter = indexer(index)
```
Comment (regadas) [inline scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala]: can we have a more explicit name? This is not really a writer per se as it is just converting arbitrary `T` to `DocWriteRequest`, wdyt?

File: scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala (hunk 3 above)
Comment (nevillelyh) [reply scio-examples/src/main/scala/com/spotify/scio/examples/extra/ElasticsearchInOut.scala]: Also the name / title is a bit misleading. We don't actually support ES input.

**Comment by KieranCTaylor** [comment]
Thanks @regadas and @nevillelyh for the review! I have made those changes now

**Comment by KieranCTaylor** [comment]
@regadas Sure, that definitely sounds more interesting! I will have a look at implementing that today

**Comment by regadas** [comment]
@KieranCTaylor great!

**Comment by KieranCTaylor** [comment]
Hey @regadas just got round to doing this (winter exams were intense), keen to know what you think!

**Comment by KieranCTaylor** [comment]
Just seen the error - will do a debug on there

File: scio-examples/src/main/scala/com/spotify/scio/examples/MinScript.scala [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+// Usage:
+
+// `sbt runMain "com.spotify.scio.examples.MinimalWordCount
+// --project=[PROJECT] --runner=DataflowRunner --zone=[ZONE]
+// --input=gs://apache-beam-samples/shakespeare/kinglear.txt
+// --output=gs://[BUCKET]/[PATH]/minimal_wordcount"`
+import com.spotify.scio._
+import com.spotify.scio.examples.common.ExampleData
+
+object MinimalWordCount {
```
Comment (regadas) [inline scio-examples/src/main/scala/com/spotify/scio/examples/MinScript.scala]: @KieranCTaylor file as added by mistake?

**Comment by KieranCTaylor** [comment]
Thanks for removing the file committed in error

**Pull request merged**